src/Morphir/IR/Package.elm

```diff
@@ -231,6 +233,51 @@ eraseDefinitionAttributes def =
         |> mapDefinitionAttributes (\_ -> ()) (\_ -> ())
 
 
+{-| Filter package by List of List of QNames
+-}
+filterPackageByTypes : List QName -> Definition ta va -> Definition ta va
+filterPackageByTypes qNames pkgDef =
```

Comment:
We are inside the `Package` module so we can omit that in function names. Knowing that it works on a definition is useful though so I would suggest `filterDefinitionByTypeNames`